pkg/asset/ignition/machine/node.go

```diff
@@ -13,6 +13,12 @@ import (
 // pointerIgnitionConfig generates a config which references the remote config
 // served by the machine config server.
 func pointerIgnitionConfig(installConfig *types.InstallConfig, rootCA []byte, role string) *ignition.Config {
+	ignitionHost := fmt.Sprintf("api-int.%s:22623", installConfig.ClusterDomain())
+	if installConfig.Platform.BareMetal != nil {
```

Comment:
i would expect a switch on platform.Name()

Reply:
Being fixed in https://github.com/openshift-metal3/kni-installer/pull/143